$OpenBSD: patch-src_calibre_ebooks_pdf_images_cpp,v 1.3 2011/11/26 19:50:08 kili Exp $

OCTStream::getRawStream deleted in poppler-0.18; Stream::getNextStream()
does the same according to the poppler documentation.

--- src/calibre/ebooks/pdf/images.cpp.orig	Fri Sep 23 19:14:20 2011
+++ src/calibre/ebooks/pdf/images.cpp	Thu Nov 17 16:12:13 2011
@@ -126,7 +126,7 @@ void XMLImages::add(GfxState *state, Object *ref, Stre
 
     if (img->type == jpeg) {
         int c;
-        str = ((DCTStream *)str)->getRawStream();
+        str = str->getNextStream();
         str->reset();
 
         // copy the stream
